app/themes/standard/views/comments/_comment_form_introduction.html.haml

```diff
@@ -7,7 +7,9 @@
     or one of your elected local councillors.
   - else
     Your comment and details will be sent to
-    %strong #{application.authority.full_name}.
-    - unless application.official_submission_period_expired?
-      They’ll consider your submission when they decide whether to approve this application.
+    %strong #{application.authority.full_name}, 
+    - if application.official_submission_period_expired?
+      but cannot be formally considered by them as the notification period for this application has expired.
```

Comment:
According to https://github.com/openaustralia/planningalerts/blob/master/spec/helpers/applications_helper_spec.rb#L160 the `on_notice_text` helper will already say "...but it will not be officially considered by the planning authority.". So, this bit seems on the face of it to be unnecessary.